airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -666,7 +666,7 @@ def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
 
 
 class PullRequestSubstream(HttpSubStream, SemiIncrementalMixin, GithubStream, ABC):
-    use_cache = False
+    use_cache = True
```

Comment:
we don't need this
we resolve caching in this PR
https://github.com/airbytehq/airbyte/pull/13558

Reply:
changed it back to false